docs/cookbook/shared-state.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Shared State
+description: How to use a reactive composable to share your objects across component files.
+author: /
+thumbnail: /recipes/animations.png
+difficulty: 0
+---
+
+# Shared State in TresJS
+
+This guide will help you get started with shared state in TresJS by building a simple scene with a cube that can be shared across component files.
+
+<StackBlitzEmbed project-id="tresjs-minimal-reproduction-rycc4j" />
```

Comment:
Hi @whitespacecode since the example doesn't rely on third party libs, can we replace this with a Tresjs repl playground https://play.tresjs.org/ ? 

You can find an example on how here 

https://github.com/Tresjs/tres/blob/8d53bbae37d858ddf775f0574647e86979473ac4/docs/cookbook/lights-shadows.md?plain=1#L14

Reply:
Hi @alvarosabu 
I tried with the playground and i'm getting this error:
`'get' on proxy: property 'modelViewMatrix' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '#<cr>' but got '#<cr>')`
I tried a few things but i'm keep getting errors on the playground.

Reply:
https://play.tresjs.org/#eNqVVelu2zgQfhVCwaIO1qYcNy12te4iG7dYtOiFpv+q/qClscWEIgWS8tHA794hdZhKL9cIYnOOb2Y+cmbuo48azH9VRTc1REk0N5nmlSUGbF0RweT6WRpZk0b/ppKXldKW3JNrtfsfVAlW78fkDZii+f9W6ZKJN8yC5kyQA1lpVZJHttAAj/4J/F3IBZMbZnqjK4uyWxNnSg9t3+kltwslrVbiO+YcpFUm8LiplwuFPyUqWmMah0JX6CBCbeDGYtI9OMUsUGnYUoCJjdM5h1RmShrnUWKxaP2sdx2de72T0w0TNaBOwtazMnI/AsZG5+NeN2RsdN7ApHIeN9eApOPBQlkJtMETIfOAvC2XudpODP8CJBPA9CRTQmm8sbO/Zs+vF0/8tRH8eK/3oE0FmeUbWLASNCMJVsktVxJdPv09Ju7vcxqRuHMbst+JSfPV2gwYdyZByOdcu4BKMvGarws7jDgdk9mYXLqICZcWJKr2qLigMxRlzNiJKRiW2Eaex8fiUTCPA2bwaOxeuJ+FLcU4lUuV78m98yuZXnOZkCnSS0jF8pzLdXcswCWWkIvp9A8v2PLcFv35kMozVlUN0C9sCVmy7G6tVS3z5ioScjadujgIg5fqE4zGUfP2JiWr6K1REvvOw6etAtstaQI6WdgaTpFGhbWVSeI4yyX65yD4RlMJNpZVGYfmV5d0Rmdxzo2NnRit0wipeQDcNNHvYDceCD+l04mEnaUXxyitdhjMboHdVUyeUkJv6wN0yJ2Qltz5hNhuwpyC6+yuEPEpph3/CaYM2cC5gP18Ms2B+dXFlD5BRGxH2NFySHJrhw9ZQ/4bwI3Dz6DRcJJDyU8A7Uxd8Zf0aSz4MoR0iPhCD/g0rcEpt+LrBw/TTUQuQL+rXPMOHygTQm1feZnVNfT5ZQVkd9+R35pdk/J7fCygN0j38SqxVcE26hc3b/FlBcpS5bVoL+cHyg9glKhdjo3ZNbYiph3Y+Wxf+j7DKfDRvNj5wdMWNUjU7ivw0k/t3Du+tUbw2RPn7f11uDn4I5aOlT2mjwPCH+6mU5fwqWvrV0vrZ7um0rzEab0BspnwFSbhQNy0VstbnOtHgZvPwTjGsr7JhVqDpfXpa2B+E42JVR9g1a92z2MaBfvWe2PenceopdSFToishRv2ThDHe1Xj2pAEJzwpsTFJk6fBwY1NLQ/nDhV2PoVVLf1mCvjoLksj71p2J/ehlDZpjnw2iOOk/gYPXwHXIhzH